packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,227 @@
+        backgroundSize: 'cover',
+      }}
+      className="flex overflow-hidden relative flex-col laptop:flex-row flex-auto items-center p-2 laptop:p-6 h-[100vh]"
+    >
+      <span className="absolute top-8 left-1/2 laptop:left-12 -translate-x-1/2 laptop:translate-x-0">
+        <Logo showGreeting={false} position={LogoPosition.Relative} />
+      </span>
+      <div className="flex z-1 flex-col p-4 laptop:p-0 laptop:mt-0 laptop:ml-3 w-full laptopL:ml-[9.315rem] laptopL:max-w-[37.2rem] mt-[5rem]">
+        <span className="flex flex-col laptop:flex-row gap-3 laptopL:gap-6 items-center laptop:items-start mb-6 tablet:mb-8">
+          <ProfileImageLink
+            user={referringUser}
+            picture={{ size: isLaptopL ? 'xxxlarge' : 'xlarge' }}
+          />
+          <p className="my-auto text-center laptop:text-left typo-headline laptop:typo-title3 laptopL:typo-title2">
+            <span className="block laptop:inline">{referringUser.name}</span>
+            <span className="font-normal"> invited you to daily.dev</span>
+          </p>
+        </span>
+        <h1 className="w-full text-center laptop:text-left break-words-overflow typo-large-title tablet:typo-mega2 laptopL:typo-giga2">
+          The homepage
+          <span className="block font-bold text-theme-color-cabbage">
+            {' '}
+            developers deserve
+          </span>
+        </h1>
+        <p className="mx-auto laptop:mx-0 mt-7 tablet:mt-8 max-w-sm laptopL:max-w-xl text-center laptop:text-left typo-title3 text-theme-label-tertiary">
+          Get a feed of the best developer news out there! Read more quality
+          articles.
+          <span className="tablet:block laptopL:inline">Stay up to date.</span>
+        </p>
+        <Button
+          buttonSize={ButtonSize.Large}
+          // important has been used is some classnames to override the default styles as agreed on [thread](https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1699023366663489?thread_ts=1699011522.350609&cid=C05P9ET7S9K)
+          className="p-4 mx-auto laptop:mx-0 mt-6 tablet:mt-12 max-w-[17.5rem] mobileL:max-w-[fit-content] btn-primary mobileL:!h-[4rem] mobileL:!p-[0] mobileL:!px-[1.5rem]"
+          tag="a"
+          href={downloadBrowserExtension}
+          onClick={() => {
+            handleAcceptClick();
+          }}
+          target="_blank"
+          rel={anchorDefaultRel}
+          loading={isLoading}
+          disabled={isLoading || isSuccess}
+        >
+          <FlexCentered className="gap-2 mobileL:typo-title3">
+            <BrowsersIcon
+              width={isMobile ? '58px' : '80px'}
+              height={isMobile ? '30px' : '40px'}
+              className="text-theme-label-primary"
+            />
+            Try it now - It&apos;s free
+          </FlexCentered>
+        </Button>
+      </div>
+      <div className="flex fixed flex-auto w-full h-full laptop:initial laptop:bottom-[unset] bottom-[-49%]">
```

Comment:
Not sure if the 1% would make a difference, but using `1/2` should result to the percentage value.
```suggestion
      <div className="flex fixed flex-auto w-full h-full laptop:initial laptop:bottom-[unset] bottom-1/2">
```

Reply:
Seems fine at 50% as well, so replacing :D